CMakeLists.txt

```diff
@@ -144,6 +148,10 @@ set(CM_PROJECTS_ALL)
 # Include all optional projects. These are the ones user can pick and choose.
 # Some of these allow using system libraries too.
 include(zlib)
+include(freetype)
+include(fontconfig)
```

Comment:
I think the change I made in the commit previously pushed is fine, and I am currently building a test on bigmac of just Qt 5 that is proceeding as expected right now.